app/lib/katello/resources/cdn.rb

```diff
@@ -1,7 +1,14 @@
 module Katello
   module Resources
     module CDN
-      SUPPORTED_SSL_VERSIONS = ['SSLv23', 'TLSv1'].freeze
+      SUPPORTED_TLS_VERSIONS_MAPPING = {
+        'TLSv1'   => OpenSSL::SSL::TLS1_VERSION,
```

Comment:
Our rubocop seems configured to not allow TableAlignment.

That's unfortunate, but not a big deal.

I'll make this line adequately ugly to satisfy rubocop so that the rest of the tests can run.